HumanControls/JoystickAxis.h

```diff
@@ -18,6 +18,6 @@ class JoystickAxis
     double calcResult(int rawValue);
 
     int m_pin;
-    double m_result, m_deadZoneSize, m_range, m_min;
+    double m_result, m_range, m_min, m_center, m_halfDeadZoneSize;
```

Comment:
I'm not sure I'd add more member variables for this. A simple divide operation could be done at the beginning of the calc function for this.